### ISSUE TITLE
Should use parseInt() for zIndex assignment

### ISSUE THREAD

**Issue by ugurarici** [alessiomaffeis/vue-picture-input#92]
When assigning zIndex values component uses zIndex+1 and zIndex+2

When use component without bindig zIndex value, it is not adding 1 and 2 to an integer, because it is a string. 

<picture-input zIndex="9"></picture-input>
Now it is a string, so "9"+1 returns 91, but we want it to be 10.

You have to bind it to keep it as a Number on component

<picture-input :zIndex="9"></picture-input>
Now it is binded as integer and 9+1 returns 10, not 91 :)

In short: We should use parseInt(zIndex)+1 for zIndex assignments :)

**Comment by alessiomaffeis** [comment]
Will be fixed in 3.0.1
